object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,16 @@
+/* exported omit */
+// Make storage for the output
+// check each value of the array and see if it matches the properties of the source.
+// if a property matches an array value, omit the properties and store the remain propeties into storage
+// check for edge cases such as null, undefined, or empty object literals.
```

Comment:
I don't see how this pseudocode matches up with the code you created at all 🤔

Reply:
i forgot to save the updated psuedo code  as i kept changing my functions for this figuring out ways that would work